tests/test_activity.py

```diff
@@ -249,6 +289,28 @@ def local_task():
             schedule_to_start + instance.timeout)
 
 
+def test_extenrnal_activity_timeouts(monkeypatch, generators):
+    """Test the creation of an external activity timeouts.
+    """
+
+    timeout = 120
+    start_timeout = 1000
+
+    monkeypatch.setattr(activity.Activity, '__init__', lambda self: None)
+    current_activity = activity.ExternalActivity(timeout=timeout)
+    current_activity.hydrate(dict(schedule_to_start=start_timeout))
+    current_activity.generators = generators
+
+    total_generators = pow(10, len(current_activity.generators))
+    schedule_to_start = start_timeout * total_generators
+    for instance in current_activity.instances({}):
+        assert current_activity.pool_size == total_generators
+        assert instance.schedule_to_start == schedule_to_start
+        assert instance.timeout == timeout
```

Comment:
Only difference with here and the one above (thus the split): timeout is static (because no tasks are provided.)